bots/run-queue

```diff
@@ -34,25 +35,34 @@ def main():
                         help='The name of the queue to consume from')
     opts = parser.parse_args()
 
-    connection = amqp.Connection(host=opts.amqp)
-    connection.connect()
+    host, port = opts.amqp.split(':')
+    connection = pika.BlockingConnection(pika.ConnectionParameters(
+        host=host,
+        port=int(port),
+        ssl=True,
+        ssl_options=pika.SSLOptions(
+            ssl_version=ssl.PROTOCOL_TLSv1_2,
+            cafile='/run/secrets/webhook/ca.pem',
+            keyfile='/run/secrets/webhook/amqp-client.key',
+            certfile='/run/secrets/webhook/amqp-client.pem',
+            server_hostname='amqp-cockpit.apps.ci.centos.org'),
```

Comment:
Same issues like in inspect-queue: Needs to support not specifying a port, and not hardcode server_hostname.